model/versioning/Versioned.php

```diff
@@ -2126,8 +2118,11 @@ public static function prepopulate_versionnumber_cache($class, $stage, $idList =
 			$parameters = $idList;
 		}
 
-		$baseClass = ClassInfo::baseDataClass($class);
-		$stageTable = ($stage == static::DRAFT) ? $baseClass : "{$baseClass}_{$stage}";
+		$baseClass = DataObject::getSchema()->baseDataClass($class);
+		$stageTable = DataObject::getSchema()->tableName($baseClass);
+		if($stage === static::LIVE) {
```

Comment:
In the prior update to ChangeSets we vetod stages other than Live or Stage. This change just cleans up some code that missed the original rewrite.

I'll update set_stage with an assert to make it clear.